src/View/Helper/HtmlHelper.php

```diff
@@ -232,7 +232,11 @@ public function meta($type, $content = null, array $options = [])
             ];
 
             if ($type === 'icon' && $content === null) {
-                $types['icon']['link'] = 'favicon.ico';
```

Comment:
@ADmad Yes, that gives the same problem. Still returns `/testing/testing` on that test case.